pulsar-client-kafka-compat/pulsar-client-kafka/src/main/java/org/apache/kafka/clients/producer/PulsarKafkaProducer.java

```diff
@@ -114,7 +114,11 @@ private PulsarKafkaProducer(Map<String, Object> conf, Properties properties, Ser
 
         String serviceUrl = producerConfig.getList(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG).get(0);
         try {
-            client = PulsarClientKafkaConfig.getClientBuilder(properties).serviceUrl(serviceUrl).build();
+            // Support Kafka's ProducerConfig.CONNECTIONS_MAX_IDLE_MS_CONFIG in ms.
+            long keepAliveIntervalMs = Long.parseLong(properties.getProperty(ProducerConfig.CONNECTIONS_MAX_IDLE_MS_CONFIG, "30000"));
+            // If passed in value is greater than Integer.MAX_VALUE in second will throw ArithmeticException.
+            int keepAliveInterval = Math.toIntExact(keepAliveIntervalMs / 1000);
```

Comment:
can you catch `ArithmeticException` and log an error message (e.g. "Invalid CONNECTIONS_MAX_IDLE_MS_CONFIG value - " + keepAliveIntervalMs + ". Please use a value smaller than " + Integer.MAX_VALUE") and then throw an InvalidArgumentException with the error message?

Reply:
Done, catch ArithmeticException, log error message and throw IllegalArgumentException with same error message.